### PR TITLE
Update .htaccess

### PR DESCRIPTION
**Pull request by MarkusSchilling**
Added RewriteRule for tensile test ontology TTO.

### CODE REVIEW

File: pmd/.htaccess [hunk 1]
```diff
@@ -7,29 +7,34 @@ RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml 
 RewriteRule ^co(.*)$ https://materialdigital.github.io/core-ontology/index-en.html [R=303,L]
 RewriteRule ^glass-ontology(.*)$ https://materialdigital.github.io/glasdigital-ontology/index-en.html [R=303,L]
+RewriteRule ^tto(.*)$ https://materialdigital.github.io/application-ontologies/tto/index-en.html [R=303,L]
```
Comment (davidlehn) [inline pmd/.htaccess:10]: I don't think this file is doing what you want, including these added rules.  The `RewriteCond` lines only effect the *next* `RewriteRule`.  They are not stateful for all following rules.  Please test and confirm.

For instance, requesting `application/ld+json` for `glass-ontology` will return the first matching rule, which is the HTML one above.
```
$ curl -sI -H "Accept: application/ld+json" https://w3id.org/pmd/glass-ontology | grep Location
Location: https://materialdigital.github.io/glasdigital-ontology/index-en.html
```

You may want to use the skip flag `S=N` "goto" trick.  There are some other hacks people have used too if you search around for multiple `RewriteRules` for one `RewriteCond`.
https://httpd.apache.org/docs/2.4/rewrite/flags.html#flag_s

**Comment by davidlehn** [comment]
@MarkusSchilling: I see you are listed as part of the listed @MaterialDigital group, so I'm assuming this is ok.  Would be good to add yourself to the README if you'll maintainer this further.

@joergwa @BerndBayerlein: You two are listed as maintainers, if you could approve this, that would be great.

**Comment by MarkusSchilling** [comment]
@davidlehn Thank you for the revision and the hints. I also changed the README.md now: I added myself as a maintainer and added the tto in the list of ontologies mentioned there. 
Furthermore, the .htaccess file was changed - thanks to @MattTJung!

**Comment by BerndBayerlein** [comment]
@Dave I can approve this

Von: David I. Lehn ***@***.***>
Gesendet: Mittwoch, 15. November 2023 23:53
An: perma-id/w3id.org ***@***.***>
Cc: Bayerlein, Bernd ***@***.***>; Mention ***@***.***>
Betreff: Re: [perma-id/w3id.org] Update .htaccess (PR #3707)

SICHERHEITSHINWEIS: Diese E-Mail wurde von außerhalb an die BAM gesendet. Bitte klicken Sie nicht auf Links oder öffnen Anhänge, bevor Sie nicht den Absender verifiziert haben und sicher sind, dass der Inhalt vertrauenswürdig ist.
This message was sent from outside of BAM. Please do not click links or open attachments unless you recognize the sender and know the content is trustworthy.


@MarkusSchilling<https://github.com/MarkusSchilling>: I see you are listed as part of the listed @materialdigital<https://github.com/materialdigital> group, so I'm assuming this is ok. Would be good to add yourself to the README if you'll maintainer this further.

@joergwa<https://github.com/joergwa> @BerndBayerlein<https://github.com/BerndBayerlein>: You two are listed as maintainers, if you could approve this, that would be great.

**Pull request merged**